website/content/contributors/getting-started.md

```diff
@@ -0,0 +1,105 @@
+
+**Design:** If you're a designer, you can contribute to open source by creating graphics, logos, or improving the website
+
+**Translation:** Many open source projects are used by people all over the world. If you're fluent in another language, you can contribute by translating the project's documentation or user interface.
+
+Regardless of your skills or background, there are many ways to contribute to open source. The key is to find a project that aligns with your interests and skill level, and to start small. Remember to read the documentation carefully and to collaborate with other contributors.
+### Understanding Git, Pull Requests, and Issues
+For both technical and non-technical contributors, it's crucial to understand Git, pull requests, and issues, even if you are not yet familiar with them. These tools are used in most open source projects to manage contributions and track changes in the code and documents. 
+
+Here's a brief overview of each:
+**Git:** Git is a version control system that allows you to track changes in your code, collaborate with others, and manage different versions of your project. You'll need to be familiar with Git and tools like GitHub, GitLab or BitBucket to contribute to open source. There is a lot of online guides available, ie.  [Learn Git with Bitbucket Cloud](https://www.atlassian.com/git/tutorials/learn-git-with-bitbucket-cloud) , 
+[Git Tutorial](https://www.w3schools.com/git/) , [Set up Git](https://docs.github.com/en/get-started/quickstart/set-up-git) and [Contributing to projects - forking](https://docs.github.com/en/get-started/quickstart/contributing-to-projects)
+
+
+#### Issues: 
+Issues are used to track bugs, feature requests, and other tasks in a project. When you find a bug or want to propose a new feature, you can create an issue in the project's issue tracker. This allows you to collaborate with other contributors and track the progress of your task. Many open source projects have "Good first issue" or "Help wanted" labels attached to certain issues, which are suitable for new contributors or those looking to make their first contribution. 
+
+These issues are typically well-defined and relatively easy to implement, and can provide a great opportunity for contributors to get started with the project and learn the workflow. Look for these labels in the issue tracker and consider starting with them if you are new to the project or open source contribution in general.
+
+#### Pull Requests: 
+Pull requests are a way to propose changes to a project. When you submit a pull request, you're asking the project maintainers to review your code or document changes and consider merging it into the main codebase. To create a pull request, you'll need to fork the project, make your changes, and then submit the pull request through the project's website or Git hosting platform.
```

Comment:
Beginners might feel upset when somebody reviews their contribution and asks for changes.

Do you think it is a good idea to mention these?
- Code reviews are essential
- Code reviews are not personal
- Code reviews are very good mechanism for learning. A beginner contributor can learn what could be potential issues with the submitted code change or if there is a better way of doing things

Reply:
Added: Code reviews are an essential part of the software development process as they help catch potential issues before they become problems. It's important to remember that code reviews are not personal and should not be taken as criticism of the developer. Instead, they provide an opportunity for developers to learn from each other and improve their skills. Code reviews can be especially beneficial for beginner contributors as they can learn from more experienced developers about potential issues with their code and how to improve it. Overall, code reviews are a valuable mechanism for learning and improving code quality. Code is not personal, but about working together to produce the best code or documentation.

Reply:
Awesome content!

Reply:
Thanks for pointing this out.